code/style.css

```diff
@@ -1,3 +1,89 @@
+* {
+  margin: 0; 
+  padding: 0;  
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: content-box;
+  -webkit-font-smoothing: antialiased; 
+  text-rendering: optimizeLegibility;
+}
+
 body {
-  background: #FFECE9;
+  background: rgb(250, 238, 231);
+  font-family: 'roboto', sans-serif;
+}
+
+.user-container img {
+  width: 250px;
+  display: grid;
+  grid-template-columns: 1, 1fr;
+  justify-items: center;
+  place-items: center;
+  margin: 15px;
+  border-style: inset;
+  border-width: 4px;
+  border-color:rgb(50, 82, 136);
+}
+
+.user-container a {
+  margin: 10px;
+  color:black;
+  text-decoration: none;
+  display: grid;
+  grid-template-columns: 1, 1fr;
+  justify-items: center;
+  place-items: center;
+  margin-bottom: 35px;
+}
+
+.repo-container {
+  display: grid;
+  grid-template-columns: 1, 1fr;
+  justify-items: center;
+  place-items: center;
+  grid-gap: 40px;
+  margin-bottom: 55px;
+}
```

Comment:
Great job with the grid-layout in the containers! I can see that a few styling/positioning things are the same for these. If you want to have a little less code you could target them all at once as well as separately. It could look something like this: ``
``suggestion

.user-container img, .user-container a, .repo-container {
  display: grid;
  grid-template-columns: 1, 1fr;
  justify-items: center;
  place-items: center;
}

.user-container img {
  width: 250px;
  margin: 15px;
  border-style: inset;
  border-width: 4px;
  border-color:rgb(50, 82, 136);
}

.user-container a {
  margin: 10px;
  color:black;
  text-decoration: none;
  margin-bottom: 35px;
}

.repo-container {
  grid-gap: 40px;
  margin-bottom: 55px;
}
```

I'm not entirely sure if this will work! But this is something that I try to learn, to target multiple elements at the same time to save some code. But maybe this suggestion would just make it harder to read the code since it is only the grid-part that are the same among the containers..